Backend/agent/handlers/medicine.py

```
@@ -0,0 +1,19 @@
+from db.db import open_db
+
+def handle(query, user_context):
```

Ruff (0.14.10)

Unused function argument: `query`

(ARG001)